tests/basic/pool_function_test.go

```diff
@@ -74,3 +83,104 @@ var _ = Describe("{StoragePoolExpandSmoky}", func() {
 		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
 	})
 })
+
+var _ = Describe("{PoolExpandWithReboot}", func() {
+	BeforeEach(func() {
+		contexts = scheduleApps()
+	})
+
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		poolToBeResized = getStoragePool(poolIDToResize)
+		storageNode, err = GetNodeWithGivenPoolID(poolIDToResize)
+		log.FailOnError(err, "Failed to get node with given pool ID")
+	})
+
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
```

Comment:
@alicelyu-px I have incorporated your changes to select pools for the tests in storage_pool_test.
can you reuse the tests PXRestartAddDisk and PoolAddDiskReboot

Reply:
https://jenkins.pwx.dev.purestorage.com/job/Users/job/alyu/job/pxdv-7-dev/35/console